django_quill/fields.py

```diff
@@ -136,13 +138,31 @@ def to_python(self, value):
         return Quill(value)
 
     def get_prep_value(self, value):
-        value = super().get_prep_value(value)
-        if value is None:
+        if value is None or isinstance(value, str):
             return value
-        if isinstance(value, Quill):
-            return value.json_string
-        return value
+        if isinstance(value, (Quill, FieldQuill)):
+            value = value.data
+
+        return json.dumps(value, cls=getattr(self, 'encoder', None))
 
     def value_to_string(self, obj):
         value = self.value_from_object(obj)
         return self.get_prep_value(value)
+
+
+class QuillTextField(QuillFieldMixin, models.TextField):
+    pass
+
+
+def QuillField(*args, **kwargs):
+    return QuillTextField(*args, **kwargs)
```

Comment:
is there a reason this returns a text field instead of a json field?

Reply:
Current users of the package, imports and uses `QuillField` which is based on text field. They may don't want to change the existing approach. It returns the text field for backward capability.

Reply:
fair point. i would just add a bit to the readme explaining the different fields so new users know what to use